setup.py

```diff
@@ -146,10 +146,10 @@ def build_extension(self, ext):
                 os.environ['_PYTHON_HOST_PLATFORM'] = re.sub(r'macosx-[0-9]+\.[0-9]+-(.+)', r'macosx-10.9-\1', util.get_platform())
 
             # Specify how many threads to use when building, depending on available memory
-            if totalMemory < 1000:
+            if freeMemory < 1000:
                 build_args += ['--', '-j1']
                 cmake_args += ['-DHUNTER_JOBS_NUMBER=1']
-            elif totalMemory < 2000:
+            elif freeMemory < 2500:
```

Comment:
Was this increase specifically tested for? Not that previous heuristics were any good, but that change will make all systems with 2GB memory build with 1 thread. I suggest adding another case and bumping down for 2 threads.

Ballpark numbers: 1 thread ~ 600-800MiB RAM

Reply:
Probably best to check on your end as well with eg, 1 or 2 threads, then calculate the number of threads for all cases based on the results you get